src/backend/src/lib.rs

```diff
@@ -112,6 +112,16 @@ pub struct State {
     user_profile_updated: UserProfileUpdatedMap,
 }
 
+impl From<&State> for Stats {
```

Comment:
Should we move this implementation into `impls` module?

Reply:
Done